python/apigw-lambda-rds/README.md

```diff
@@ -0,0 +1,44 @@
+# Api Gateway with Lambda & RDS (Mysql)
+
+This example creates an API Gateway with a Lambda that will query data from the RDS Mysql Instance. 
+
+After the RDS Instance is provisioned a Cloudformation custom resource is used to create a DB populate with some example data.
+
+The lambda function & custom resource use Secrets Manager for storage & retrieval of DB connection details.
+
+## Prep
+
+Both of the lambda functions require python modules to be installed prior to deployment.
+
+In this examples root folder, run the following:
+
+```bash
+pip install -r lambda/requirements.txt -t lambda/
+pip install -r cfn-lambda/requirements.txt -t cfn-lambda/
```

Comment:
might be helpful to add a prefix that indicates what the lambda does if it can be expressed succinctly